config/config.exs

```diff
@@ -10,6 +10,8 @@ import Config
 config :franklin,
   ecto_repos: [Franklin.Repo]
 
+config :franklin, Franklin.Repo, migration_timestamps: [type: :utc_datetime]
```

Comment:
This will change the default behavior of `timestamps()`, but as you'll see in our migration files, we are still preferring an explicit use of `utc_datetime`.